test/functional/feature_segwit.py

```diff
@@ -48,17 +48,20 @@ def set_test_params(self):
                 "-rpcserialversion=0",
                 "-vbparams=segwit:0:999999999999",
                 "-addresstype=legacy",
+                "-txindex"
```

Comment:
I'm still not thrilled about adding `txindex` to a bunch of tests. See my comment here: https://github.com/bitcoin/bitcoin/pull/13931/files#r216084729

Reply:
I added a todo / note on this particular file & am happy to work on updating tests in a future PR. 
To clarify, this PR adds `txindex` to many tests. Are you commenting on the set as a whole, or specifically these `feature_segwit` tests?

Reply:
I added a todo / note on this particular file & am happy to work on updating tests in a future PR.
To clarify, this PR adds txindex to many tests. Are you commenting on the set as a whole, or specifically these feature_segwit tests?

Reply:
I would add a short comment to each of the tests. Something like "txindex required for getrawtransaction call"

Reply:
I added todos to the tests. Not my favorite to be littering the code, but IMO a comment without the todo makes it seem intentional instead of transient.